Source/options.js

```diff
@@ -512,6 +512,16 @@ $(function() {
     save('ngramMax', 'Maximum word sequence length saved');
   }
 
+  function validate_yandexTranslatorApiKey(){
+    var yandexKey = e("yandexTranslatorApiKey").value
+    if (yandexKey.match(/Unblock|Block/)){
+      alert("Please enter valid Yandex key.\nRemove Block key / Unblock key from the end of the key")
```

Comment:
If they are copying and pasting a key with "Unblock key" then that key is blocked. Seems like a separate exceptional case.